### DISCUSSION TITLE
Fuxa running  in Linux Mint problem  with plugin "node-snap7"

### DISCUSSION THREAD

**Discussion by mrHubdoc** [frangoteam/FUXA#948] [Q&A]
Have installed with node js Version 14.21.3. gets Fuxa to start but it says "[WAR] try to create 314 but plugin is missing!
" Can't communicate with the PLC.

Have tried to install on a Windows 10 machine then I get contact with the PLC.

**Comment by unocelli** [comment]
Hi, I have just added a description (WARNING) for installation in linux in the README

**Comment by mrHubdoc** [comment]
I noted that warning. I think I have tried to follow those instructions. But there is no guarantee that it will work anyway?

**Reply by unocelli** [reply]
I don't understand, have you tried?

**Reply by mrHubdoc** [reply]
Yes I have tried

**Reply by unocelli** [reply]
And, is working? I'm sorry but I don't understand your questions? If you don't install the package you will not be able to create the device for SiemensS7 communications.

**Reply by mrHubdoc** [reply]
Have installed the correct package.

**Reply by unocelli** [reply]
FUXA have 3 type of installation, which one did you use? try via npm.

**Reply by mrHubdoc** [reply]
npm install -g --unsafe-perm @frangoteam/fuxa
fuxa

**Reply by unocelli** [reply]
you must install from source code, and remove the node-snap7 package from the server/package.json before installing

**Reply by mrHubdoc** [reply]
I want to use "node-snap7" Is it possible in linux?

**Reply by unocelli** [reply]
There is on the github FUXA home page! we didn't hide it or?

WARNING In linux with nodejs Version 16.20.2 || 18.18.2 there are problems with the node-snap7 library for communication with Siemens S7 PLCs, if you don't intend to use it you can remove it from the server/package.json

Install from [NPM](https://www.npmjs.com/package/@frangoteam/fuxa) (first option) In linux to use only with nodejs Version 14.21.3

**Reply by mrHubdoc** [reply]
I have done my best to follow the instructions. How can I troubleshoot?
